### PR TITLE
Feature/dif modify/pelagos 3324 add unit tests for dif entity

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

**Comment by snguyen1** [comment]
This is good! Should we wait for the migration pull request to be merged then write test function for theApprovedDate for this pull request?

**Comment by praneethpr** [comment]
@snguyen1 Either way I'm okay, merge this and then i can update that add attribute branch or merge that pull request, i can update this branch.

File: tests/unit/Pelagos/Entity/DifTest.php [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+     */
+    protected $dif;
+
+    /**
+     * @var ResearchGroup
+     */
+    protected $mockResearchGroup;
+
+    /**
+     * Setup for all the test cases in National Data center entity test.
```
Comment (mickel1138) [inline tests/unit/Pelagos/Entity/DifTest.php]: I guess this test came from National Data center!? ;-)

Comment (praneethpr) [reply tests/unit/Pelagos/Entity/DifTest.php]: Haha.. not the entire test  :P I think just the setup

**Comment by fightingtexasaggie** [comment]
https://proteus.tamucc.edu/~mwilliamson/coverage-3324/mwilliamson/pelagos/src/Pelagos/Entity/DIF.php.html
Here's the current coverage, for reference. Eventually we'll get to 100% on all entities ;-)

**Pull request merged**